drivers/storage/portworx/components_test.go

```diff
@@ -3937,13 +3937,55 @@ func TestCSIInstallWithNewerCSIVersion(t *testing.T) {
 	require.NoError(t, err)
 	require.Empty(t, csiDriver.OwnerReferences)
 	require.False(t, *csiDriver.Spec.AttachRequired)
-	require.False(t, *csiDriver.Spec.PodInfoOnMount)
+	require.True(t, *csiDriver.Spec.PodInfoOnMount)
 
 	csiDriver = &storagev1beta1.CSIDriver{}
 	err = testutil.Get(k8sClient, csiDriver, pxutil.DeprecatedCSIDriverName, "")
 	require.True(t, errors.IsNotFound(err))
 }
 
+func TestCSIInstallWithK8s120Version(t *testing.T) {
```

Comment:
Does this have a dependency on k8s version or just px version needs to be above 2.5? 
If only dependent on Portworx, should we rename the test to `TestCSIInstallWithPx2_5Version`?

Reply:
CSI ephemeral support depends on Portworx & K8s version. I seemed to have messed up the PR breakdown, let me fix that.